src/main/views/sscs-daily-list.njk

```diff
@@ -36,12 +36,16 @@
     <summary class="govuk-details__summary"><span class="govuk-details__summary-text">{{ importantInformationHeading }}</span></summary>
     <div class="govuk-details__text">
       <p>{{ importantInformationP1 }}</p>
-      <p>{{ importantInformationP2 | replace("NAME", courtName) | replace("EMAIL", listData['venue']["venueContact"]["venueEmail"]) | replace("TELNO", listData['venue']['venueContact']['venueTelephone'])}}</p>
+      <p>{{ importantInformationP2 | replace("NAME", courtName) | replace("EMAIL", listData['venue']["venueContact"]["venueEmail"]) | replace("TELNO", listData['venue']['venueContact']['venueTelephone']) }}</p>
```

Comment:
I've spotted venueEmail and venueTelephone is optional in the schema, has this been discussed at all?

If not I think it's worth raising, as I guess we'd end up with blanks in the open justice statement

Reply:
Spoke with Chris Carter. Will create a separate ticket and make venueEmail and venueTelephone mandatory in the schema